x-pack/plugins/security_solution/cypress/integration/host_details/risk_tab.spec.ts

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
I would move the whole spec file to the hosts folder instead of creating a new one.